go/src/koding/klientctl/team.go

```diff
@@ -3,15 +3,63 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-	"koding/klientctl/endpoint/team"
 	"os"
+	"text/tabwriter"
+
+	"koding/kites/kloud/team"
+	epteam "koding/klientctl/endpoint/team"
 
 	"github.com/codegangsta/cli"
 	"github.com/koding/logging"
 )
 
+func TeamList(c *cli.Context, log logging.Logger, _ string) (int, error) {
+	opts := &epteam.ListOptions{
+		Slug: c.String("slug"),
+	}
+
+	teams, err := epteam.List(opts)
+	if err != nil {
+		return 0, err
+	}
+
+	if len(teams) == 0 {
```

Comment:
No, It's not opposite to what we have now. Eg. you handle NotFound errors in `credential.list` on [Kloud side](https://github.com/koding/koding/blob/bd496ac0a9c293791db54bbb89576bef6969fe5a/go/src/koding/kites/kloud/credential/mongo.go#L393). Thus, you return empty(non nil slice) and "ignores" NotFound error in Kloud. This is exactly what I'm doing here.